Src/Support/Google.Apis.Core/Http/ConfigurableMessageHandler.cs

```diff
@@ -72,6 +72,11 @@ public class ConfigurableMessageHandler : DelegatingHandler
         /// </summary>
         public const string CredentialKey = "__CredentialKey";
 
+        /// <summary>
+        /// Key for a universe domain in a <see cref="HttpRequestMessage"/> options.
+        /// </summary>
+        public const string UniverseDomainKey = "__UniverseDomainKey";
```

Comment:
Does this need to be public?

Reply:
Would prefer to "default to internal" in general. We've definitely made more things public here than was probably wise (and the Google.Apis vs Google.Apis.Core split doesn't help; something we should consider addressing if we ever create a new major version) but I think it's worth trying to stop adding extra public things that don't need to be public.